systems/frontier_user_guide.rst

```diff
@@ -248,6 +248,7 @@ A single Frontier node contains 4 MI250X OAMs for the total of 8 GCDs.
     Therefore, from this point on, the Frontier guide simply refers to a GCD as a GPU.
 
 Each GPU contains 110 Compute Units (CUs) grouped in 4 Compute Engines (CEs).
+Physically, each GPU contains 112 CUs, but two are disabled for yield reasons.
```

Comment:
Hey Jakub, can you clarify what you mean when you say yield reasons? I'm not sure that's clear.

Reply:
Okay, I removed "yield reasons".
There's no good reason confusing the reader with semiconductor manufacturing issues.
Seems beyond the scope of the guide.